babel.config.json

```diff
@@ -11,8 +11,10 @@
         "@babel/proposal-optional-chaining"
       ]
     }],
-    "@babel/preset-react",
+    ["@babel/react", { "runtime": "automatic" }],
```

Comment:
This will use the new jsx "runtime"
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html